library/core/src/main/java/com/google/android/exoplayer2/source/SilenceMediaSource.java

```diff
@@ -33,6 +33,30 @@
 /** Media source with a single period consisting of silent raw audio of a given duration. */
 public final class SilenceMediaSource extends BaseMediaSource {
 
+  public static final class Factory {
```

Comment:
Please provide a short java doc for public methods which are not overridden. Here and below.